x-pack/plugins/ingest_manager/server/services/epm/registry/requests.ts

```diff
@@ -81,3 +85,44 @@ function isFetchError(error: FailedAttemptErrors): error is FetchError {
 function isSystemError(error: FailedAttemptErrors): boolean {
   return isFetchError(error) && error.type === 'system';
 }
+
+type ProxyAgent = IHttpsProxyAgent | HttpProxyAgent;
+export function getFetchOptions(url: string): RequestInit | {} {
+  // start with env values and can add from flags, defaults, etc later
+  const proxyUrl = getProxyFromEnvForUrl(url);
+  if (!proxyUrl) {
+    return {};
+  }
+
+  const logger = appContextService.getLogger();
+  logger.info(`Using proxy ${proxyUrl} from environment variable`);
```

Comment:
Is this logged every single time fetch is used?

Reply:
Yes, but I downgraded it to a `debug` message 

```typescript
logger.debug(`Using ${proxyUrl} from environment variable as proxy for ${targetUrl}`);
```

Reply:
Not for this PR, but I suggest we move this logging to an initial setup place where it is logged once that the proxy is used and on the info level.